impexp-config/src/main/java/org/citydb/config/project/kmlExporter/GltfOptions.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * 3D City Database - The Open Source CityGML Database
+ * http://www.3dcitydb.org/
+ *
+ * Copyright 2013 - 2019
+ * Chair of Geoinformatics
+ * Technical University of Munich, Germany
+ * https://www.gis.bgu.tum.de/
+ *
+ * The 3D City Database is jointly developed with the following
+ * cooperation partners:
+ *
+ * virtualcitySYSTEMS GmbH, Berlin <http://www.virtualcitysystems.de/>
+ * M.O.S.S. Computer Grafik Systeme GmbH, Taufkirchen <http://www.moss.de/>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.citydb.config.project.kmlExporter;
+
+import javax.xml.bind.annotation.XmlAttribute;
+import javax.xml.bind.annotation.XmlType;
+import java.io.File;
+import java.util.Locale;
+
+@XmlType(name = "GltfOptionsType", propOrder = {
+        "pathToConverter",
+        "gltfVersion",
+        "removeColladaFiles",
+        "embedTextures",
+        "useBinaryGltf",
+        "useDracoCompression"
+})
+public class GltfOptions {
+    @XmlAttribute
+    private boolean createGltfModel;
+    private String pathToConverter;
+    private GltfVersion gltfVersion;
+    private Boolean removeColladaFiles;
+    private Boolean embedTextures;
+    private Boolean useBinaryGltf;
+    private Boolean useDracoCompression;
+
+    public GltfOptions() {
+        gltfVersion = GltfVersion.v2_0;
+        embedTextures = true;
+        useDracoCompression = true;
+
+        pathToConverter = "contribs" + File.separator + "collada2gltf";
+        String osName = System.getProperty("os.name").toLowerCase(Locale.ENGLISH);
+        if (osName.contains("windows"))
+            pathToConverter += File.separator + "COLLADA2GLTF-v2.1.3-windows-Release-x64" + File.separator + "COLLADA2GLTF-bin.exe";
```

Comment:
v 2.15 of COLLADA2glTF is available: https://github.com/KhronosGroup/COLLADA2GLTF/releases/tag/v2.1.5

Reply:
Yes, we have already tested with version 2.1.5 but it fails when using Draco compression. 2.1.4 seems to work though. We will have to do more tests to confirm.

Reply:
Updated COLLADA2glTF converter to v2.1.4 with 89a163d6326dc4d03e5cf6345b8e67f41338d60d. Further tests confirmed that version 2.1.5 has issues when using Draco compression.